webpack.config.js

```diff
@@ -97,6 +100,7 @@ HashBundlePlugin.prototype.apply = function f(compiler) {
 if (!isProd) {
   config.plugins.push(new DashboardPlugin());
 } else {
+  delete config.devtool;
```

Comment:
I think it's more confusing to set things specifically for dev vs. for prod. But we can do it the other way. Let's merge this asap though and open an issue so I can use debugger :)